src/work-allocation/release2/components/available-tasks-filter/available-tasks-filter.component.spec.ts

```diff
@@ -0,0 +1,155 @@
+import { Location } from '@angular/common';
```

Comment:
This looks essentially identical to the original component. Why is this needed? Do we simply have a series of cloned components in here?

Reply:
yep. this was brought up a couple of times in meetings, whether we should create new components by extending from existing components or modifying existing ones to have addditional functionality. in the end we are duplicating but only what we need. down the road im sure the "release 1"  versions will becomes obsolete.